api/fossa/tar.go

```diff
@@ -363,12 +368,16 @@ func tarballUpload(name string, dependency, upload bool, tarball *os.File, hash
 		return Locator{}, err
 	}
 
-	dependencyParameter := url.Values{}
+	parameters := url.Values{}
 	if dependency {
-		dependencyParameter.Add("dependency", "true")
+		parameters.Add("dependency", "true")
+	}
 
+	if rawLicenseScan {
+		parameters.Add("rawLicenseScan", "true")
```

Comment:
Is the backend wired up for this?

Reply:
Looks like there's a pending PR for that. This PR looks good, but in your place I'd make sure the API on the server side is finalized before you merge this.